### PR TITLE
[zh] Localize the decision tree for priorityclass blog

### PR DESCRIPTION
**Pull request by wilsonwu**
Localize the decision tree for priorityclass blog:
```
content/zh-cn/blog/_posts/2023-01-12-protect-mission-critical-pods-priorityclass/decision-tree.svg
```

### CODE REVIEW

File: content/zh-cn/blog/_posts/2023-01-12-protect-mission-critical-pods-priorityclass/decision-tree.svg
Comment (sftim) [inline content/zh-cn/blog/_posts/2023-01-12-protect-mission-critical-pods-priorityclass/decision-tree.svg:1]: nit: you could write “PriorityClass” (singular) in the box that is top right.

**Review by sftim — COMMENTED** [review]
Thanks for the PR

File: content/zh-cn/blog/_posts/2023-01-12-protect-mission-critical-pods-priorityclass/decision-tree.svg
Comment (wilsonwu) [reply content/zh-cn/blog/_posts/2023-01-12-protect-mission-critical-pods-priorityclass/decision-tree.svg:1]: Done

**Comment by windsonsea** [comment]
Keep a proper para spacing as the origin en paras:

```
根据 PriorityClass 值，调度程序将所有 Pod 放入队列中。

dev-nginx 最低，而 prod-nginx 最高。
```
State a fact in past not a future behavior:
```
在我们的例子中，因为没有足够的资源，dev-nginx 被驱逐了。
```
nit:
```
在我们的例子中，prod-nginx 被成功调度到 node01。
```
Some word `gracefully` is missing:
```
最低优先级的 Pod 被优雅终止，为高优先级的 Pod 腾出空间。
```

**Comment by wilsonwu** [comment]
> Keep a proper para spacing as the origin en paras:
> 
> ```
> 根据 PriorityClass 值，调度程序将所有 Pod 放入队列中。
> 
> dev-nginx 最低，而 prod-nginx 最高。
> ```
> 
> State a fact in past not a future behavior:
> 
> ```
> 在我们的例子中，因为没有足够的资源，dev-nginx 被驱逐了。
> ```
> 
> nit:
> 
> ```
> 在我们的例子中，prod-nginx 被成功调度到 node01。
> ```
> 
> The word `gracefully` is missing:
> 
> ```
> 最低优先级的 Pod 被优雅终止，为高优先级的 Pod 腾出空间。
> ```

All fixed, but the first suggest has a little problem, if keep 2 lines, the content will be out of the box boundary, so keep them in 4 lines

**Comment by tengqm** [comment]
overall looks good.
"terminated gracefully" 建议翻译为“体面地终止”，
一个 Pod 被终止，很难做到动作优雅地死去，比较体面地死去是有可能的。

**Comment by wilsonwu** [comment]
All Done.

**Pull request merged**